File: Ohtap E Commerce Domain Models/Development/PHP/Objects/PackageLabels/IPackageLabelData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\PackageLabels;


use  Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface IPackageLabelData extends IPackageLabel,IPackageLabelValue,CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{

}

File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/PackageLabels/IPackageLabelData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.PackageLabels
{
    public interface IPackageLabelData :  CoreObject.IObjectID, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState, IPackageLabelValue, IPackageLabelLabel
    {

    }
}
